docs/pages/setup/guides/teleport-operator.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Teleport Kubernetes Operator (Preview)
+description: Easily manage Teleport resources from Kubernetes
+---
+
+<Admonition type="warning" title="Preview">
+  Teleport Kubernetes Operator is currently in Preview mode.
+</Admonition>
+
+The Teleport Kubernetes Operator provides a way for Kubernetes users to manage some Teleport resources through
+Kubernetes, following [the Operator Pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+
+The Teleport Kubernetes Operator is deployed alongside its CustomResourceDefinitions. Once deployed, users
+can use a Kubernetes client like `kubectl` or their existing CI/CD Kubernetes pipelines to create Teleport
+CustomResources. The Teleport Kubernetes Operator watches for those resources and does API calls to Teleport to
+reach the desired state.
+
+Currently supported Teleport resources are:
+- users
+- roles
+
+This guide covers how to:
+- Install Teleport Operator on Kubernetes.
+- Configure Teleport users and roles using Kubernetes.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- Kubernetes cluster (with or without `teleport-cluster` already deployed);
```

Comment:
I would say "teleport-cluster Helm chart", otherwise not clear what teleport-cluster is referring to for new users.